sentry/content.md

```diff
@@ -8,19 +8,41 @@ Sentry is a realtime event logging and aggregation platform. It specializes in m
 
 # How to use this image
 
-## start a sentry instance
+## how to setup a full sentry instance
 
-### PostgreSQL database (as recommended by upstream)
+1.	start a redis container
 
-	docker run --name some-sentry --link some-postgres:postgres -d sentry
+		docker run -d --name some-redis redis
 
-### MySQL database
+2.	start a MySQL or PostgreSQL(recommended by upstream) database container (later steps assume PostgreSQL, replace the `--link` entries appropriately for MySQL)
 
-	docker run --name some-sentry --link some-mysql:mysql -d sentry
+		# postgres:
+		docker run -d -e POSTGRES_PASSWORD=secret -e POSTGRES_USER=sentry --name some-postgres postgres
+		# or mysql:
+		docker run -d -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=sentry --name some-mysql mysql
```

Comment:
If you change these to a sub-list, you can make this slightly easier to look at: (and the intent of that initial sentence easier to grok, especially at a glance)
```markdown
2.	start a database container

	-	PostgreSQL: (recommended by upstream)

			docker run -d -e POSTGRES_PASSWORD=secret -e POSTGRES_USER=sentry --name some-postgres postgres

	-	MySQL: (later steps assume PostgreSQL, replace the later `--link` entries appropriately for MySQL)

			docker run -d -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=sentry --name some-mysql mysql
```

Rendered:

2.	start a database container

	-	PostgreSQL: (recommended by upstream)

			docker run -d -e POSTGRES_PASSWORD=secret -e POSTGRES_USER=sentry --name some-postgres postgres

	-	MySQL: (later steps assume PostgreSQL, replace the later `--link` entries appropriately for MySQL)

			docker run -d -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=sentry --name some-mysql mysql